src/Storages/ObjectStorage/StorageObjectStorageSink.h

```diff
@@ -50,6 +50,7 @@ class PartitionedStorageObjectStorageSink : public PartitionedSink
         const ASTPtr & partition_by);
 
     SinkPtr createSinkForPartition(const String & partition_id) override;
+    SinkPtr createSinkForHivePartition(const std::string & partition_id) override;
```

Comment:
This extra method might not be needed if I can get rid of https://github.com/ClickHouse/ClickHouse/pull/76802/files#diff-f5bc37f041316e685e98f6069ffee32fc67704092df667292b5bd6aeccd6611fR124

Reply:
Actually, I'll make things simpler. I'll just mimic the existing behavior